src/test/java/io/confluent/connect/jdbc/JdbcSourceConnectorTest.java

```diff
@@ -139,6 +139,8 @@ public void testStartStop() throws Exception {
     // Close with stopping will be invoked when the connector is stopped
     mockCachedConnectionProvider.close(true);
     PowerMock.expectLastCall().atLeastOnce();
+    mockCachedConnectionProvider.close();
```

Comment:
`close(true)` already calls `close()`.  Why is it being called explicitly again?

Reply:
close(true) checks for call with `true` parameter passed. There is another thread in the connector which calls the `close()` method and due to timing issues this test had become flaky